fission-web-server/library/Fission/Web/Server/Types.hs

```diff
@@ -441,19 +441,14 @@ instance App.Domain.Initializer Server where
 instance App.Content.Initializer Server where
   placeholder = asks appPlaceholder
 
-instance JWT.Resolver Server where
+instance UCAN.Resolver Server where
   resolve cid =
-    IPFS.ipfsCat cid >>= \case
+    IPFS.ipfsCat cid <&> \case
```

Comment:
`<&>` has caused a lot of grief over the years. We standardized to `>>=`, since it's easy to miss the `<&>` and wonder why the typechecker is complaining. Plus GHC will do this rewrite at compile time for us.

Historically _I_ have used a lot of `<&>` in this exact scenario, but it's probably best avoided. I think maybe stylish-haskell recommends these eta reductions, and we should probably configure that out.

Reply:
It's actually `hlint` usually suggesting things, but I've turned it off for me.
Yeah, I was thinking "we can get rid of two `return`s here", but I agree that's really not worth it and going for a monadic lambda makes sense generally! :+1: